### PR TITLE
Docs: Instructions to separate emails with semicolons

### PR DESCRIPTION
**Pull request by mjseaman**
**What this PR does / why we need it**:
Adds instructions for how to enter multiple emails into the report recipients field

**Which issue(s) this PR fixes**:
None - submitting PR directly.

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by osg-grafana** [comment]
What is the rationale for a given syntax? Would this ever be something copied and pasted from a CSV-based source?

File: docs/sources/enterprise/reporting.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ Currently only Organization Admins can create reports.
 1. Enter report information. All fields are required unless otherwise indicated.
    - **Name -** Name of the report as you want it to appear in the Reports list.
    - **Source dashboard -** Select the dashboard to generate the report from.
-   - **Recipients -** Enter the emails of the people or teams that you want to receive the report.
+   - **Recipients -** Enter the emails of the people or teams that you want to receive the report, separated by semicolons.
```
Comment (osg-grafana) [inline docs/sources/enterprise/reporting.md:32]:
```suggestion
   - **Recipients -** Enter the emails of the people or teams that you want to receive the report, separated by commas.
```

**Review by osg-grafana — CHANGES_REQUESTED** [review]
Rationale for semicolon non-existent.

File: docs/sources/enterprise/reporting.md (hunk 1 above)
Comment (mjseaman) [reply docs/sources/enterprise/reporting.md:32]: Semicolons are the **current** functionality, so we need to document it even though we intend to change it.

Comment (osg-grafana) [reply docs/sources/enterprise/reporting.md:32]: By "intend to change it", do you mean for a given release?

Comment (mjseaman) [reply docs/sources/enterprise/reporting.md:32]: Yes, I think [grafana-enterprise #1210](https://app.zenhub.com/workspaces/grafana-enterprise-5f7f23d9e9cc5c0012cbcb07/issues/grafana/grafana-enterprise/1210) will be released in v8.0, but in the meantime we need to document the current functionality, which is that email addresses need to be separated by semicolons. 

An example of why this should be documented: https://raintank-corp.slack.com/archives/C017631SZFW/p1617092786122600

**Review by osg-grafana — APPROVED** [review]
Approving as is.

**Comment by osg-grafana** [comment]
@mjseaman, you want to resign that CLA and see if there is anything needed for this issue? It otherwise looks like it can be merged. WDYT?

**Comment by torkelo** [comment]
this change wont make it into latest docs or v8 docs without a backport label to the corresponding release branches

**Comment by achatterjee-grafana** [comment]
Adding the backport label. @torkelo, do we need to backport to get information into 'next`, now that we are in beta 2? Thank you.

**Pull request merged**